### PR TITLE
Update census household schema

### PR DESCRIPTION
**Pull request by bitdivision**
### What is the context of this PR?
We need to update the census household schema in runner to include the latest changes.

Since the schema is not being maintained, there is no need for the cypress functional test anymore. Given that we are very likely to reuse cypress in future, I have left the infrastructure around it in there for the moment.

The schema that is being copied is available here: https://github.com/ONSdigital/eq-census-schemas/blob/census_household_hannah_2/census_household.json as per the trello card: https://trello.com/c/iqEpCo7q/2813-update-census-household-in-runner

### How to review 
Make sure the census household schema works

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: scripts/run_tests_functional.sh [hunk 1]
```diff
@@ -34,7 +34,3 @@ yarn test_functional
 
 display_result $? 5 "Front end functional tests"
 
-echo "Running census functional tests"
-yarn test_census
```
Comment (MebinAbraham) [inline scripts/run_tests_functional.sh:38]: May as well remove it from the README also.

Comment (bitdivision) [reply scripts/run_tests_functional.sh:38]: I've just changed it to test_cypress for the moment, since the functionality is still there.

**Review by MebinAbraham — APPROVED** [review]
Matches the other schema so I guess 👍

**Comment by iwootten** [comment]
I get a fair amount of what looks like formatting differences between what's linked to in the trello card after it's been formatted (maybe worth linking that gist here too) and the schema in the PR here.

**Comment by bitdivision** [comment]
@iwootten Yep, I did have to reformat it as we lint the schemas. I probably should have done that as two commits sorry.

**Comment by bitdivision** [comment]
If you wanted to double check it, you could copy the schema, run `yarn format` and diff the two.

**Comment by iwootten** [comment]
@bitdivision that's after me having run through the gulp format - should there be differences after that? I wonder if we could automate linting of the schemas in the census household repo in the future.

**Comment by bitdivision** [comment]
@iwootten Ah, right. I'll have a look. There was only one material change for a bad ID I believe.

I'd rather just get rid of that repo. I don't think CI linting will work well unless we can run `yarn format` and auto commit the result (since it's committed through the web interface)

**Comment by bitdivision** [comment]
@iwootten You're right. There are some formatting differences. I can only assume that was down to pycharm and the formatter doesn't fully manage it. 

I've updated the schema and avoided using pycharm.

**Review by iwootten — APPROVED** [review]

**Pull request merged**